src/components/controls/selects/SearchSelect/SearchSelect.stories.tsx

```diff
@@ -82,16 +106,58 @@ const InputValueTemplate: ComponentStory<typeof SearchSelect<(typeof inputValueO
   )
 }
 
-export const InputValue = InputValueTemplate.bind({})
-InputValue.args = {
-  label: "Label",
-  options: inputValueOptions,
-}
-
-export const CustomFilter = Template.bind({})
+export const CustomFilter = DescriptionTemplate.bind({})
 CustomFilter.args = {
   ...Basic.args,
+  options: descriptionOptions,
   filter: (value, option) =>
     option.value.toLowerCase().trim().includes(value.toLowerCase().trim()) ||
     option.description.toLowerCase().trim().includes(value.toLowerCase().trim()),
 }
+CustomFilter.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement)
+  const inputField = canvas.getByRole("textbox")
+  inputField.focus()
```

Comment:
In the spirit of making the tests resemble how users use the software as much as possible, I think a click/tab to focus is preferable over programmatic focus. But fine either way!

```suggestion
  await userEvent.click(label)
```